src/Tester/Tester.csproj

```diff
@@ -43,62 +43,67 @@
     <AssemblyOriginatorKeyFile>..\Orleans.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bond">
-      <HintPath>$(SolutionDir)packages\Bond.Runtime.CSharp.3.0.7\lib\net45\Bond.dll</HintPath>
```

Comment:
I think it is actually the other way around....

To ensure quality software, software engineers really should ask [and be able to answer] why **every** line of change in a PR / change set is needed.

Using an excuse of "a dev tool told me to do it" is the fastest way to bit-rot that i know!

The corollary is also true - if a particular changed line is not **directly** related to the PR change in hand, then it really should be split out into a separate PR / change set.
You will frequently see @gabikliot say exactly that during code review: "please submit [this part] as a separate PR"

By keeping change sets small and focused, it is much easier to isolate any problems that may be introduced along the way, even if later on we have to go back and exclude change requests one-by-one to see exactly what caused the issue to start.

Thankfully, tools like `SourceTree` make it really easy to obsess about and manage code quality down to this line-by-line level.